src/utils/config_utils/src/config_utils.c

```diff
@@ -204,9 +205,9 @@ static void ADUC_AgentInfo_Free(ADUC_AgentInfo* agent)
  * @param agentCount Count of objects in agents.
  * @param agents Array of ADUC_AgentInfo objects to free.
  */
-static void ADUC_AgentInfoArray_Free(size_t agentCount, ADUC_AgentInfo* agents)
+static void ADUC_AgentInfoArray_Free(unsigned int agentCount, ADUC_AgentInfo* agents)
 {
-    for (size_t index = 0; index < agentCount; ++index)
+    for (unsigned int index = 0; index < agentCount; ++index)
```

Comment:
seems like this entire file can be reverted back to commit before branch point - git checkout <commit> -- <path/to/file>
same for config_utils.h